src/client/datascience/jupyterDebugService.ts

```diff
@@ -252,6 +253,27 @@ export class JupyterDebugService implements IJupyterDebugService, IDisposable {
         this.onClose();
     }
 
+    public requestKernelDebugAdapterVariables(msg: DebugProtocolMessage): void {
+        switch ((msg as any).type) {
+            case 'request':
+                this.debugAdapterTrackers.forEach((d) => {
```

Comment:
I don't understand why you need to forward messages to the debug adapter trackers manually, isn't vscode doing that when you register them for the session?

Reply:
we ended up removing this